src/server/src/test/resources/io.cassandrareaper.acceptance/integration_reaper_functionality.feature

```diff
@@ -415,16 +415,26 @@ Feature: Using Reaper
     Then reaper has no longer the last added cluster in storage
   ${cucumber.upgrade-versions}
 
-@sidecar
+  @sidecar
+  @current_test
```

Comment:
I think you probably want to remove this before merging.

Reply:
I totally do 👍